cucumber/api/features/policy_load_errors.feature

```diff
@@ -61,6 +67,13 @@ Feature: Policy loading error messages
       }
     }
     """
+    And there is an audit record matching:
+    """
+      <85>1 * * conjur * policy
+      [auth@43868 user="cucumber:user:admin"][subject@43868]
+      [action@43868 result="failure" operation="add"]
+      [meta sequenceId="4"] Failed to load policy: policy_text resource has a blank id
```

Comment:
```suggestion
      [meta sequenceId="4"] Failed to load policy: policy_text resource is empty (has a blank ID)
```

Reply:
This message has to match to a message that comes from other parties, so it cannot be modified